salt/states/boto_ec2.py

```diff
@@ -1752,5 +1749,169 @@ def private_ips_absent(name, network_interface_name=None, network_interface_id=N
 
         # there were no changes since we did not attempt to remove ips
         ret['changes'] = {}
+    return ret
+
+
+def secgroup_present(
```

Comment:
I'm just saying we don't want the helpers to be the primary mechanism for managing the groups.